lib/embercatcher_api_web/controllers/session_controller.ex

```diff
@@ -0,0 +1,27 @@
+defmodule EmbercatcherApiWeb.SessionController do
+  use EmbercatcherApiWeb, :controller
+
+  alias EmbercatcherApi.Account
+  alias EmbercatcherApi.Account.User
+
+  action_fallback EmbercatcherApiWeb.FallbackController
+
+  def create(conn, %{"session" => session_params}) do
+    with %User{} = user <- Account.get_user_by_email!(session_params["email"]),
```

Comment:
This will raise an error if a user's email isn't found. I think the default here would be that Phoenix sends a 400. But you might want to make a separate function to get a user by email and not raise when an email isn't found. That way you can control the flow more and return a 401.